packages/components/constraints/src/horizontal/horizontal.story.js

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
-import { withKnobs, select } from '@storybook/addon-knobs/react';
+import { withKnobs } from '@storybook/addon-knobs/react';
 import { customProperties as vars } from '@commercetools-uikit/design-system';
 import styled from '@emotion/styled';
 import Horizontal from './horizontal';
 import Readme from '../../README.md';
 
 const ColouredRow = styled.div`
-  height: 100px;
+  height: 30px;
```

Comment:
maybe document (code comment) why this is 30px?

Reply:
It's just for the storybook example, so it's a totally arbitrary value.
In this case, I just changed it to allow all sizes to fit on a standard Mac screen.